packages/mobile/package-lock.json

```diff
@@ -9972,7 +9972,7 @@
     "absolute-path": {
       "version": "0.0.0",
       "resolved": "https://registry.npmjs.org/absolute-path/-/absolute-path-0.0.0.tgz",
-      "integrity": "sha512-HQiug4c+/s3WOvEnDRxXVmNtSG5s2gJM9r19BTcqjp7BWcE48PB+Y2G6jE65kqI0LpsQeMZygt/b60Gi4KxGyA=="
+      "integrity": "sha1-p4di+9rftSl76ZsV01p4Wy8JW/c="
```

Comment:
😢 try nuking node modules first? these shouldn't be flipping to sha1

Reply:
rip i merged without seeing this, will evert lol

Reply:
I think it's fine - another branch with nuked node modules will flip em all back to 512. We can focus on the fix and come back to clean up